CONTRIBUTING.md

```diff
@@ -42,70 +42,22 @@ Merging Pull Requests (for project contributors with write access)
 
 ### Pre-requisites
 
-This project contains Git submodules which are required to build. After cloning this repo, run `git submodule update
---init --recursive`.
-
-### CMake
-CMake version 3.12 or higher is required to build these libraries. Download and install CMake from the project's
+#### CMake
+CMake version 3.15 or higher is required to build these libraries. Download and install CMake from the project's
 [website](https://cmake.org/download/).
 
-### Vcpkg
-Vcpkg is required to download project dependencies. To get started, first clone vcpkg to a location on your system and
-run the bootstrapping script.
+#### Vcpkg
+Vcpkg can be used to have the Azure for SDK for CPP dependencies installed into a specific folder in the system instead of globally installing them.
+Follow [vcpkg install guide](https://github.com/microsoft/vcpkg) to have next dependencies installed
```

Comment:
```suggestion
Follow [vcpkg install guide](https://github.com/microsoft/vcpkg) to install the following dependencies:
```

Reply:
What install guide are we linking to? Linking to the vcpkg repo isn't helpful.

Isn't it just clone vcpkg, bootstrap, and install? We should list the steps right in this doc, including checking out the specific tag/versions we support
https://github.com/Azure/azure-sdk-for-cpp/blob/e3503f8575cc67d5663683071755d0ea1aff5625/eng/vcpkg-commit.txt#L1

Reply:
We haven't set a tag for what we want to support on cpp.
And I don't want to provide the VCPKG instructions because it can change. That's VCPKG who owns this guide.
I can try to provide a link to a readme section

Reply:
> We haven't set a tag for what we want to support on cpp.

We haven't?! That's what we use in CI and what we should document as supported (just like Embedded C):
https://github.com/Azure/azure-sdk-for-cpp/blob/2cc4ecdea8404b98d418a7bd37fa5c62d4d377ea/eng/pipelines/templates/steps/vcpkg.yml#L13

> And I don't want to provide the VCPKG instructions because it can change. 

Then why did we provide it in Embedded C? How likely is that to happen? I think the less links-out of the docs, the easier it is for the reader to just follow the steps and have them work.

Reply:
Let's close on this.